stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/MessagesHeaderViewModel.kt

```diff
@@ -14,11 +14,11 @@ import io.getstream.chat.android.livedata.ChatDomain
 /**
  * ViewModel class for [com.getstream.sdk.chat.view.ChannelHeaderView].
  * Responsible for updating channel information.
- * Can be bound to the view using [ChannelHeaderViewModel.bindView] function.
+ * Can be bound to the view using [MessagesHeaderViewModel.bindView] function.
  * @param cid the full channel id, i.e. "messaging:123"
  * @param chatDomain entry point for all livedata & offline operations
  */
-public class ChannelHeaderViewModel @JvmOverloads constructor(
+public class MessagesHeaderViewModel @JvmOverloads constructor(
```

Comment:
This is a breaking change. Are we sure we want to do that?

Reply:
But this is inside `stream-chat-android-ui-common`, so there are no no users for the module yet.

Reply:
`stream-chat-android-ui-common` is shared between the old SDK `stream-chat-android` and the new one. So whoever uses it will be forced to change ViewModel's name

Reply:
Indeed, I thought it was `stream-chat-android-ui-components`... You're right, it is a breaking change.

Reply:
I'll revert the breaking change and we can discuss those renaming problems in the proper meeting.